src/renderer/modals/RepairModal/index.js

```diff
@@ -133,13 +133,32 @@ type Props = {
 class RepairModal extends PureComponent<Props, *> {
   state = {
     selectedOption: null,
+    availableRepairChoices: [
+      ...repairChoices,
+      { forceMCU: "", label: "Something else", id: "somethingElse" },
```

Comment:
Wouldn't this make it not translatable?

Reply:
Yes - but I thought that all the labels from `repairChoices`, coming from LLC, were not translated. But actually, i just saw that there is a translation with key `somethingElse` so I'm going to change for this :clock1:

Reply:
Updated using a translation key:

<details>
  <summary>New translated label</summary>

![Screenshot from 2022-04-21 09-40-21](https://user-images.githubusercontent.com/15096913/164405090-1efaae34-8fdc-47ae-919c-d282da002ff8.png)

</details>

Reply:
So nothing else is translated :wtf:

Reply:
Yes, for this old modal for Nano S, the labels text are coming from [deviceWordings.ts on LLC](https://github.com/LedgerHQ/ledger-live-common/blob/develop/src/deviceWordings.ts) - and used [here in firmwareUpdate-repair.ts](https://github.com/LedgerHQ/ledger-live-common/blob/d63df6a2fadeb0e2c1d49823ec5ed91062d58f11/src/hw/firmwareUpdate-repair.ts#L22)